packages/components/src/button/stories/best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta } from '@storybook/blocks';
+import FigmaEmbed from '/storybook/components/figma-embed';
+
+<Meta title="Components/Actions/Button/Best Practices" />
+
+# Button
+Buttons allow users to take actions and make selections with a single click or tap.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-280&node-type=frame&t=JLKM1grrtWleRkFR-11" title="kitchen-sink-button" height="400" />
+
+
+## Usage
+Buttons indicate available actions and allow user interaction within the interface. As key elements in the WordPress UI, they appear in toolbars, modals, and forms. Default buttons support most actions, while primary buttons emphasize the main action in a view. Secondary buttons pair as secondary actions next to a primary action. 
+
+Each layout contains one prominently placed, high-emphasis button. If you need multiple buttons, use one primary button for the main action, secondary for the rest of the actions and tertiary sparingly when an action needs to not stand out at all.
+
+## Variants
+
+### Primary
+Primary buttons stand out with bold color fills, making them distinct from the background. Since they naturally draw attention, each layout should contain only one primary button to guide users toward the most important action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-640&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="primary" height="400" />
+
+### Secondary
+Secondary buttons complement primary buttons. Use them for standard actions that may appear alongside a primary action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-808&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="secondary" height="400" />
+
+### Tertiary
+Tertiary buttons have minimal emphasis. Use them sparingly to subtly highlight an action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-874&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="tertiary" height="400" />
+
+### Default
+Use default buttons for actions of equal priority.
```

Comment:
I still do not understand when we should use `tertiary` vs. `default` 😅 @WordPress/gutenberg-design Is there any more specific guidance we can give, or maybe an example of when they should be used?

Reply:
I don't understand this either. My take is we address this when we take a look at button philosophy. (Unless someone knows a fundamental difference between the two) 😅

Reply:
Wonderful discussion. For me it's about the names themselves, primary, secondary, tertiary suggest a hierarchy where none exists. There isn't a specific rule of "Tertiary buttons must be next to secondary buttons", etc. I'd much more have appreciated simple button names: solid, outline, text, for which we could then provide very simple instructions for each:

* Use a solid button when you want to convey prominence of an action, that the action is important or primary for the context.
* Use an outline button for lower prominence, or when the context benefits from the outline aligning with other bordered elements.
* Use a text button for lowest prominence: these are for less important actions.

Reply:
Is there an existing issue for button design system work where we can add these thoughts as feedback? cc @jasmussen @jameskoster

Reply:
There's #63856 which is probably the main one.